requirements.txt

```diff
@@ -1,9 +1,12 @@
+attrs==19.1.0
 certifi==2019.3.9
 chardet==3.0.4
 idna==2.8
 jsonschema==3.0.1
 kinto-http==10.3.0
+pyrsistent==0.14.11
 requests==2.21.0
 ruamel.yaml==0.15.90
+six==1.12.0
```

Comment:
Why is six still required? Which package pulls this?

Reply:
`Collecting six>=1.11.0 (from jsonschema->kinto-wizard==4.1.0.dev0)`